diff --git a/Cargo.toml b/Cargo.toml
index bd49d23..880308b 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,8 +1,7 @@
 [package]
-authors = ["{{authors}}"]
 edition = "2018"
 readme = "README.md"
-name = "{{project-name}}"
+name = "cortex-m-quickstart"
 version = "0.1.0"
 
 [dependencies]
@@ -24,12 +23,6 @@ panic-halt = "0.2.0"
 # features = ["stm32f303", "rt"]
 # version = "0.7.1"
 
-# this lets you use `cargo fix`!
-[[bin]]
-name = "{{project-name}}"
-test = false
-bench = false
-
 [profile.release]
 codegen-units = 1 # better optimizations
 debug = true # symbols are nice and they don't increase the size on Flash
